package.json

```diff
@@ -318,6 +318,18 @@
                     "default": true,
                     "description": "[DEPRECATED] Specifies whether code completion should provide snippets (true) or plain-text items (false).",
                     "deprecationMessage": "Use 'kotlin.completion.snippets.enabled'"
+                },
+                "kotlin.watchFiles": {
```

Comment:
I'm a bit torn on whether we should name this `kotlin.watchFiles` or `kotlin.languageServer.watchFiles`. On the one hand, it doesn't related _directly_ to the execution of the language server, on the other hand, all "meta options" relating to the language server are prefixed with `kotlin.languageServer` (e.g. the port, transport, ...). The options under `kotlin` are generally intended for handling by the language server directly (this might be a questionable convention, but we should stay consistent where possible).